docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs.md

```diff
@@ -0,0 +1,150 @@
+
+When you create a Microsoft Exchange Trace Logs Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure a Microsoft Exchange Trace Logs Source:
+
+1.In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+
+2.On the Collectors page, click **Add Source** next to a Hosted Collector.
+
+3.Select **MS Exchange Trace Logs**.
+
+    <img src={useBaseUrl('img/send-data/ms-exchange-icon.png')} alt="ms-exchange-icon.png" width="200" />
+
+4.Enter a **Name** for the Source. The description is optional.
+
+    <img src={useBaseUrl('img/send-data/ms-exchange-schema-ui.png')} alt="ms-exchange-schema-ui.png" width="400" />
+
+5.(Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+
+6.(Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate, each field needs a name (key) and value.
+
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema it is ignored, known as dropped.
+
+8.**Application (client) ID**. Enter your client ID from your Azure Application. This should be a Globally Unique Identifier aka GUID.
+
+9.**Directory (tenant) ID**. Enter your tenant ID from your Azure Application. This should be a Globally Unique Identifier aka GUID.
+
+10.**Secret**. Enter your client secret generated within your Azure Application.
+
+11.**Oauth 2.0 Authorization Code** Enter a valid authorization code. They expire within 1 hour of creation. The Microsoft Exchange Trace Logs Source will use this temporary code to automatically generate and rotate your API tokens for accessing the data.
+
+12.**Polling Interval Seconds**. We recommend leaving this to its defaults. This value controls how often the Microsoft Exchange Trace Logs Source polls the the Microsoft APIs for new data.
+
+13.**Collection Time Offset**. Use this value to offset the collection in the past by some amount of time. Microsoft claims data can be missing from the API within the last 24 hours. Set this to `86400` seconds aka 24 hours to help guarantee more complete results or reduce it if you are looking for more real time, but potentially incomplete data.
+
+14.When you are finished configuring the Source, click **Submit**.
```

Comment:
please close gaps between steps (delete lines in between)

Reply:
DONE